packages/near-membrane-base/src/environment.ts

```diff
@@ -1,192 +1,181 @@
                 continue;
             }
-            if (!canRedPropertyBeTamed(redValue, key)) {
-                // eslint-disable-next-line no-console
-                console.warn(`Property ${String(key)} of ${redValue} cannot be remapped.`);
-                // eslint-disable-next-line no-continue
-                continue;
-            }
             // Avoid poisoning by only installing own properties from blueDescriptors
-            // @ts-expect-error because PropertyDescriptorMap does not accept symbols ATM.
+            // @ts-ignore
             const blueDescriptor = { __proto__: null, ...blueDescriptors[key] };
-            const redDescriptor = { __proto__: null, ...blueDescriptor };
-            if ('value' in blueDescriptor) {
-                redDescriptor.value = broker.getRedValue(blueDescriptor.value);
-            } else {
-                // Use the original getter to return a red object, but if the
-                // sandbox attempts to set it to a new value, this mutation will
-                // only affect the sandbox's global object, and the getter will
-                // start returning the new provided value rather than calling onto
-                // the blue realm. This is to preserve the object graph of the
-                // blue realm.
-                // istanbul ignore next
-                let currentBlueGetter: (this: RedValue) => RedValue = () => undefined;
-
-                if (typeof blueDescriptor.get === 'function') {
-                    const { get: blueGetter } = blueDescriptor;
-                    // Note: The reason why we don't use broker.getRedValue here is because we
-                    // want that proxy to be lazy. This brings other questions: what about error
-                    // control? Do we have test for this? Can we optimize this so after the
-                    // first call we don't pay the cost of wrapping anymore?
-                    //
-                    // TODO: Isn't it easier to just not do any lazy stuff anymore considering
-                    // that the creation of those proxies is now faster?
-                    const blueDistortedGetter = this.distortionCallback(
-                        blueGetter
-                    ) as () => BlueValue;
-                    currentBlueGetter = function currentDistortedBlueGetter() {
-                        const value: BlueValue = ReflectApply(
-                            blueDistortedGetter,
-                            broker.getBlueValue(this),
-                            emptyArray
-                        );
-                        return broker.getRedValue(value);
-                    };
-                    redDescriptor.get = function get(): RedValue {
-                        return ReflectApply(currentBlueGetter, this, emptyArray);
-                    };
-                }
+            const configurable = ObjectHasOwnProperty.call(blueDescriptor, 'configurable')
```

Comment:
Because `blueDescriptor` has a `null` proto you skip the `ObjectHasOwnProperty` check and do the `in` check for partial descriptor stuff.